locales/base/translation.json

```diff
@@ -1230,6 +1230,7 @@
   "merchantPaymentSubmitSuccess": "Your payment successfully submitted",
   "merchantWalletUnlockError": "Something went wrong when unlocking your wallet, please try again",
   "noMoreTransactions": "There are no more transactions",
+  "quoteUpdateFailed": "Could not refresh the quote, please retry",
```

Comment:
is this copy from figma? I think we consistently use "try again" instead of "retry" in other error messages